CMakeLists.txt

```diff
@@ -259,6 +259,9 @@ ADD_DEFINITIONS(-D_USE_MATH_DEFINES)
 # Qt 5.7 requires C++11 support
 SET(STEL_GCC_C_FLAGS "-Wall -Wextra -Wno-unused-parameter -Wno-unknown-pragmas")
 SET(STEL_GCC_CXX_FLAGS "-Wall -Wextra -Wno-unused-parameter -Wno-unknown-pragmas")
+# LTO support
+SET(STEL_GCC_C_FLAGS "${STEL_GCC_C_FLAGS} -flto -Werror=odr -Werror=lto-type-mismatch -Werror=strict-aliasing")
```

Comment:
I'd leave these default ones (in CMakeLists) without Werror, but just Wodr, etc (though they might already be enabled via Wall Wextra)

Werror can be added in CI